vendors/swedbank-pay-sdk-php/tests/bootstrap.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-checkout/pullRequest?prid=5510424)